### ISSUE TITLE
Matrix CI fails as expected - fix once we start supporting Py3.12 and beyond

### ISSUE THREAD

**Issue by bobleesj** [regro/regolith#1215]
### Problem

<img width="591" alt="Screenshot 2024-12-12 at 11 08 58 AM" src="https://github.com/user-attachments/assets/5336cd18-3ffe-467b-9be6-da89a8487c29" />

Shown above, since it's recently cookiecuttered, the reusable CI workflow is used. However, due to the constraint that we `pip install gooey` in the CI, the reusable workflow can't be used.

Furthermore, `regolith` currently supports `py311` hence we should re-visit once our package support py312 ad beyond?

**Comment by sbillinge** [comment]
@bobleesj @tinatn29 I will try and get a new version of Gooey out to conda-forge before this release, bu tbecause wx is now on cf at 3.13 I think we can do a 3.13 release even if I don't have gooey done.  @bobleesj I could use some help with Gooey maybe.

**Comment by bobleesj** [comment]
@sbillinge yes, happy to help with gooey conda forge 3.13. I assume the gooey author has granted admin access for the feedstock?

**Comment by sbillinge** [comment]
closed by #1294
